ee/codegen/src/__test__/__snapshots__/graph-attribute.test.ts.snap

```diff
@@ -174,6 +174,25 @@ exports[`Workflow > graph > should be correct for two branches merging from sets
 "
 `;
 
+exports[`Workflow > graph > should be pointing to the correct terminal nodes from a nested set of conditionals 1`] = `
```

Comment:
Manually edited this snapshot to match the intended target

Reply:
Note that there are actually three valid graphs possible. The set for `FinalCheckNode` could go in any of the three spots where its referenced